README.md

```diff
@@ -30,23 +30,14 @@ git checkout <s2n-commit-hash-used-by-aws-crt-cpp>
 # We keep the build artifacts in the -build directory
 cd libcrypto-build
 
-# Download the latest version of OpenSSL
-curl -LO https://www.openssl.org/source/openssl-1.1.1-latest.tar.gz
-tar -xzvf openssl-1.1.1-latest.tar.gz
-
-# Build openssl libcrypto.  Note that the install path specified here must be absolute.
-cd `tar ztf openssl-1.1.1-latest.tar.gz | head -n1 | cut -f1 -d/`
-./config -fPIC no-shared              \
-         no-md2 no-rc5 no-rfc3779 no-sctp no-ssl-trace no-zlib     \
-         no-hw no-mdc2 no-seed no-idea enable-ec_nistp_64_gcc_128 no-camellia\
-         no-bf no-ripemd no-dsa no-ssl2 no-ssl3 no-capieng                  \
-         -DSSL_FORBID_ENULL -DOPENSSL_NO_DTLS1 -DOPENSSL_NO_HEARTBEATS      \
-         --prefix=<absolute-install-path>
-make
-make install
+# Get the latest version of AWS-LC
```

Comment:
These instructions clone aws-lc into `<root>/s2n/libcrypto-build/aws-lc`

Should we have users build aws-lc **before** s2n, and just have them put aws-lc in their root, like we do with otehr aws-c-* dependencies?

Reply:
Also, we tell users to checkout "s2n-commit-hash-used-by-aws-crt-cpp"
Should we also tell users to checkout a particular aws-lc hash?

Reply:
I think that makes sense, and I can update the location where `aws-lc` is cloned into. I was just trying to keep it similar to the current process when using OpenSSL. 

Also, we are discussing using a tagged release when checking out and building `aws-lc`. I can update that once I have a better answer after that meeting.

Reply:
The current directions for building s2n are super confusing, I would not feel beholden to them.

A straightahead series of:
clone X, install X, clone Y, install Y, clone Z, install Z, ...
seems best to me